CMake/ITKModuleClangFormat.cmake

```diff
@@ -0,0 +1,15 @@
+# Download and configure clang-format for style uniformity during development.
+#
+# This code helps download a version of clang-format binaries consistent with
+# ITK's clang-format configuration. Git configuration is added during
+# development to help automatically apply the formatting.
+#
+# Globally, the ITK/.clang-format file provides the base style configuration.
+# For more information on ITK coding style, see the ITK Coding Style Guide in
+# the ITK Software Guide.
+option(ITK_USE_CLANG_FORMAT "Enable the use of clang-format for coding style formatting." ${BUILD_TESTING})
```

Comment:
@thewtex Should this be `ITK_USE_CLANGFORMAT`, cross-referencing
https://github.com/InsightSoftwareConsortium/ITK/blob/master/CMake/ITKClangFormatSetup.cmake#L11
or else, should the later be renamed to `ITK_USE_CLANG_FORMAT`?

I've just realized that both options are available in CMake, and I am not sure about their difference. Thanks

Reply:
I think one option is for downloading it, and the other for using it.

Reply:
> I think one option is for downloading it, and the other for using it.

Then maybe the variable names and their descriptions should honor better their purpose?

Reply:
I am not entirely sure of the distinction. If you could double-check it, feel free to update the names and descriptions.

Reply:
@jhlegarreta good catch! They were intended to be the same. Patch here:https://github.com/InsightSoftwareConsortium/ITK/pull/2594

Reply:
Thanks both for caring about this. Resolving.